include/tvc.h

```diff
@@ -60,38 +72,61 @@ extern char __LIB__ fputc_cons_native(int character);
  * earlier this call will block (func $A1)
  */
 #define tvc_kbd_chin     fgetc_cons
-extern int __LIB__ fgetc_cons();
+int __LIB__ fgetc_cons();
```

Comment:
Is this one needed? cf getk() comment previously

Reply:
Nope, removed.